girder/cli/mount.py

```diff
@@ -49,6 +49,8 @@ class ServerFuse(fuse.Operations):
     extended to expose metadata and other resources by extending the available
     paths.  Files can also be reached via a path shortcut of /file/<id>.
     """
+    use_ns = True
```

Comment:
Should this be grouped with the other attributes below?

Reply:
Conceptually they are different.  The other are specifically file system methods we don't want to handle.  This is a flag for fusepy's Operations class.